src/fsharp/fsi/fsi.fs

```diff
@@ -1362,6 +1406,14 @@ module internal MagicAssemblyResolution =
 
         let rangeStdin = rangeN Lexhelp.stdinMockFilename 0
 
+#if TODO_REWORK_ASSEMBLY_LOAD
+        ignore tcConfigB
+        ignore tcImports
+        ignore fsiDynamicCompiler
+        ignore fsiConsoleOutput
+        ignore rangeStdin
+        ()
+#else
```

Comment:
I don't think so, although I am considering asking for one on the new Loader.  However, I don't expect we'll need one, once we have fixed our assembly resolution mechanism.  It is essentially a hack.